main.go

```diff
@@ -192,13 +194,25 @@ func keygenCmd(c *cli.Context) error {
 	if !args.Present() {
 		slog.Fatal("Missing drand address in argument (IPv4, dns)")
 	}
+
+	addr := args.First()
+	var validID = regexp.MustCompile(`[:][0-9]+$`)
```

Comment:
Let's keep the port given by the user between 2 and 5 chars. Always validate as much as possible any user input data.